### PR TITLE
python312Packages.langgraph*: 20241206 update

### PR DESCRIPTION
**Pull request by sarahec**
## Changes

1. Bump to latest releases
 - `langgraph`: 0.2.43 -> 0.2.56
 - `langgraph-sdk`: 0.1.35 -> 0.1.43
 - `langgraph-checkpoint`: 2.0.2 -> 2.0.8
 - `langgraph-checkpoint-postgres`: 2.0.2 -> 2.0.8
2. Update `langgraph-checkpoint-postgres` derivation to work with newly revised unit tests.
3. Update `langgraph-checkpoint-{sqlite,postgresql,duckdb}` clients to accept current `langgraph-checkpoint`. (Sqlite and duckdb adaptors haven't been updated in a while.)

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [x] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [x] `sandbox = true`
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

File: pkgs/development/python-modules/langgraph-checkpoint-postgres/default.nix [hunk 1]
```diff
@@ -46,23 +47,41 @@ buildPythonPackage rec {
...
     pytest-asyncio
     pytestCheckHook
+    postgresqlTestHook
+    (postgresql.withPackages (p: [ p.pgvector ]))
   ];
 
-  passthru = {
-    updateScript = langgraph-sdk.updateScript;
+  env = {
+    postgresqlTestUserOptions = "LOGIN SUPERUSER";
```
Comment (drupol) [inline pkgs/development/python-modules/langgraph-checkpoint-postgres/default.nix:65]: I'm unsure about this, shouldn't this be only made during the tests ?

Comment (sarahec) [reply pkgs/development/python-modules/langgraph-checkpoint-postgres/default.nix:65]: This variable is only used by the postgresqlCheckHook and this is a fairly common way to use it. I could set it in `preCheck` if you like.

Comment (drupol) [reply pkgs/development/python-modules/langgraph-checkpoint-postgres/default.nix:65]: I think it would be a better idea.

WDYT @GaetanLepage ?

Comment (sarahec) [reply pkgs/development/python-modules/langgraph-checkpoint-postgres/default.nix:65]: Give me a sec, I'll do it.

Comment (sarahec) [reply pkgs/development/python-modules/langgraph-checkpoint-postgres/default.nix]: Moved into `preCheck` (which clarifies its use better, thank you).

Comment (GaetanLepage) [reply pkgs/development/python-modules/langgraph-checkpoint-postgres/default.nix]: Good for me ! `preCheck` makes sense indeed.

**Review by drupol — APPROVED** [review]
Diff LGTM

**Comment by GaetanLepage** [comment]
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 362648`

---
### `x86_64-linux`
<details>
  <summary>:x: 8 packages failed to build:</summary>
  <ul>
    <li>python311Packages.langgraph</li>
    <li>python311Packages.langgraph-checkpoint-postgres</li>
    <li>python311Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python311Packages.langgraph.dist</li>
    <li>python312Packages.langgraph</li>
    <li>python312Packages.langgraph-checkpoint-postgres</li>
    <li>python312Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python312Packages.langgraph.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 16 packages built:</summary>
  <ul>
    <li>python311Packages.langgraph-checkpoint</li>
    <li>python311Packages.langgraph-checkpoint-duckdb</li>
    <li>python311Packages.langgraph-checkpoint-duckdb.dist</li>
    <li>python311Packages.langgraph-checkpoint-sqlite</li>
    <li>python311Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python311Packages.langgraph-checkpoint.dist</li>
    <li>python311Packages.langgraph-sdk</li>
    <li>python311Packages.langgraph-sdk.dist</li>
    <li>python312Packages.langgraph-checkpoint</li>
    <li>python312Packages.langgraph-checkpoint-duckdb</li>
    <li>python312Packages.langgraph-checkpoint-duckdb.dist</li>
    <li>python312Packages.langgraph-checkpoint-sqlite</li>
    <li>python312Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python312Packages.langgraph-checkpoint.dist</li>
    <li>python312Packages.langgraph-sdk</li>
    <li>python312Packages.langgraph-sdk.dist</li>
  </ul>
</details>

---
### `aarch64-linux`
<details>
  <summary>:x: 8 packages failed to build:</summary>
  <ul>
    <li>python311Packages.langgraph</li>
    <li>python311Packages.langgraph-checkpoint-postgres</li>
    <li>python311Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python311Packages.langgraph.dist</li>
    <li>python312Packages.langgraph</li>
    <li>python312Packages.langgraph-checkpoint-postgres</li>
    <li>python312Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python312Packages.langgraph.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 16 packages built:</summary>
  <ul>
    <li>python311Packages.langgraph-checkpoint</li>
    <li>python311Packages.langgraph-checkpoint-duckdb</li>
    <li>python311Packages.langgraph-checkpoint-duckdb.dist</li>
    <li>python311Packages.langgraph-checkpoint-sqlite</li>
    <li>python311Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python311Packages.langgraph-checkpoint.dist</li>
    <li>python311Packages.langgraph-sdk</li>
    <li>python311Packages.langgraph-sdk.dist</li>
    <li>python312Packages.langgraph-checkpoint</li>
    <li>python312Packages.langgraph-checkpoint-duckdb</li>
    <li>python312Packages.langgraph-checkpoint-duckdb.dist</li>
    <li>python312Packages.langgraph-checkpoint-sqlite</li>
    <li>python312Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python312Packages.langgraph-checkpoint.dist</li>
    <li>python312Packages.langgraph-sdk</li>
    <li>python312Packages.langgraph-sdk.dist</li>
  </ul>
</details>

---
### `x86_64-darwin`
<details>
  <summary>:white_check_mark: 24 packages built:</summary>
  <ul>
    <li>python311Packages.langgraph</li>
    <li>python311Packages.langgraph-checkpoint</li>
    <li>python311Packages.langgraph-checkpoint-duckdb</li>
    <li>python311Packages.langgraph-checkpoint-duckdb.dist</li>
    <li>python311Packages.langgraph-checkpoint-postgres</li>
    <li>python311Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python311Packages.langgraph-checkpoint-sqlite</li>
    <li>python311Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python311Packages.langgraph-checkpoint.dist</li>
    <li>python311Packages.langgraph-sdk</li>
    <li>python311Packages.langgraph-sdk.dist</li>
    <li>python311Packages.langgraph.dist</li>
    <li>python312Packages.langgraph</li>
    <li>python312Packages.langgraph-checkpoint</li>
    <li>python312Packages.langgraph-checkpoint-duckdb</li>
    <li>python312Packages.langgraph-checkpoint-duckdb.dist</li>
    <li>python312Packages.langgraph-checkpoint-postgres</li>
    <li>python312Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python312Packages.langgraph-checkpoint-sqlite</li>
    <li>python312Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python312Packages.langgraph-checkpoint.dist</li>
    <li>python312Packages.langgraph-sdk</li>
    <li>python312Packages.langgraph-sdk.dist</li>
    <li>python312Packages.langgraph.dist</li>
  </ul>
</details>

---
### `aarch64-darwin`
<details>
  <summary>:white_check_mark: 24 packages built:</summary>
  <ul>
    <li>python311Packages.langgraph</li>
    <li>python311Packages.langgraph-checkpoint</li>
    <li>python311Packages.langgraph-checkpoint-duckdb</li>
    <li>python311Packages.langgraph-checkpoint-duckdb.dist</li>
    <li>python311Packages.langgraph-checkpoint-postgres</li>
    <li>python311Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python311Packages.langgraph-checkpoint-sqlite</li>
    <li>python311Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python311Packages.langgraph-checkpoint.dist</li>
    <li>python311Packages.langgraph-sdk</li>
    <li>python311Packages.langgraph-sdk.dist</li>
    <li>python311Packages.langgraph.dist</li>
    <li>python312Packages.langgraph</li>
    <li>python312Packages.langgraph-checkpoint</li>
    <li>python312Packages.langgraph-checkpoint-duckdb</li>
    <li>python312Packages.langgraph-checkpoint-duckdb.dist</li>
    <li>python312Packages.langgraph-checkpoint-postgres</li>
    <li>python312Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python312Packages.langgraph-checkpoint-sqlite</li>
    <li>python312Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python312Packages.langgraph-checkpoint.dist</li>
    <li>python312Packages.langgraph-sdk</li>
    <li>python312Packages.langgraph-sdk.dist</li>
    <li>python312Packages.langgraph.dist</li>
  </ul>
</details>

**Comment by GaetanLepage** [comment]
`langgraph-checkpoint-postgres` fails with:
```
Running phase: pytestCheckPhase
Executing pytestCheckPhase
/nix/store/m1p78gqlc0pw3sdbz3rdhklzm0g26g96-stdenv-linux/setup: line 227: export: `=': not a valid identifier
/nix/store/m1p78gqlc0pw3sdbz3rdhklzm0g26g96-stdenv-linux/setup: line 227: export: `LOGIN SUPERUSER': not a valid identifier
```

**Comment by sarahec** [comment]
@ofborg build python312Packages.langgraph-checkpoint-postgres

**Comment by GaetanLepage** [comment]
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 362648`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 24 packages built:</summary>
  <ul>
    <li>python311Packages.langgraph</li>
    <li>python311Packages.langgraph-checkpoint</li>
    <li>python311Packages.langgraph-checkpoint-duckdb</li>
    <li>python311Packages.langgraph-checkpoint-duckdb.dist</li>
    <li>python311Packages.langgraph-checkpoint-postgres</li>
    <li>python311Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python311Packages.langgraph-checkpoint-sqlite</li>
    <li>python311Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python311Packages.langgraph-checkpoint.dist</li>
    <li>python311Packages.langgraph-sdk</li>
    <li>python311Packages.langgraph-sdk.dist</li>
    <li>python311Packages.langgraph.dist</li>
    <li>python312Packages.langgraph</li>
    <li>python312Packages.langgraph-checkpoint</li>
    <li>python312Packages.langgraph-checkpoint-duckdb</li>
    <li>python312Packages.langgraph-checkpoint-duckdb.dist</li>
    <li>python312Packages.langgraph-checkpoint-postgres</li>
    <li>python312Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python312Packages.langgraph-checkpoint-sqlite</li>
    <li>python312Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python312Packages.langgraph-checkpoint.dist</li>
    <li>python312Packages.langgraph-sdk</li>
    <li>python312Packages.langgraph-sdk.dist</li>
    <li>python312Packages.langgraph.dist</li>
  </ul>
</details>

---
### `aarch64-linux`
<details>
  <summary>:x: 10 packages failed to build:</summary>
  <ul>
    <li>python311Packages.langgraph</li>
    <li>python311Packages.langgraph-checkpoint</li>
    <li>python311Packages.langgraph-checkpoint-duckdb</li>
    <li>python311Packages.langgraph-checkpoint-duckdb.dist</li>
    <li>python311Packages.langgraph-checkpoint-postgres</li>
    <li>python311Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python311Packages.langgraph-checkpoint-sqlite</li>
    <li>python311Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python311Packages.langgraph-checkpoint.dist</li>
    <li>python311Packages.langgraph.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 14 packages built:</summary>
  <ul>
    <li>python311Packages.langgraph-sdk</li>
    <li>python311Packages.langgraph-sdk.dist</li>
    <li>python312Packages.langgraph</li>
    <li>python312Packages.langgraph-checkpoint</li>
    <li>python312Packages.langgraph-checkpoint-duckdb</li>
    <li>python312Packages.langgraph-checkpoint-duckdb.dist</li>
    <li>python312Packages.langgraph-checkpoint-postgres</li>
    <li>python312Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python312Packages.langgraph-checkpoint-sqlite</li>
    <li>python312Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python312Packages.langgraph-checkpoint.dist</li>
    <li>python312Packages.langgraph-sdk</li>
    <li>python312Packages.langgraph-sdk.dist</li>
    <li>python312Packages.langgraph.dist</li>
  </ul>
</details>

---
### `x86_64-darwin`
<details>
  <summary>:white_check_mark: 24 packages built:</summary>
  <ul>
    <li>python311Packages.langgraph</li>
    <li>python311Packages.langgraph-checkpoint</li>
    <li>python311Packages.langgraph-checkpoint-duckdb</li>
    <li>python311Packages.langgraph-checkpoint-duckdb.dist</li>
    <li>python311Packages.langgraph-checkpoint-postgres</li>
    <li>python311Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python311Packages.langgraph-checkpoint-sqlite</li>
    <li>python311Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python311Packages.langgraph-checkpoint.dist</li>
    <li>python311Packages.langgraph-sdk</li>
    <li>python311Packages.langgraph-sdk.dist</li>
    <li>python311Packages.langgraph.dist</li>
    <li>python312Packages.langgraph</li>
    <li>python312Packages.langgraph-checkpoint</li>
    <li>python312Packages.langgraph-checkpoint-duckdb</li>
    <li>python312Packages.langgraph-checkpoint-duckdb.dist</li>
    <li>python312Packages.langgraph-checkpoint-postgres</li>
    <li>python312Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python312Packages.langgraph-checkpoint-sqlite</li>
    <li>python312Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python312Packages.langgraph-checkpoint.dist</li>
    <li>python312Packages.langgraph-sdk</li>
    <li>python312Packages.langgraph-sdk.dist</li>
    <li>python312Packages.langgraph.dist</li>
  </ul>
</details>

---
### `aarch64-darwin`
<details>
  <summary>:white_check_mark: 24 packages built:</summary>
  <ul>
    <li>python311Packages.langgraph</li>
    <li>python311Packages.langgraph-checkpoint</li>
    <li>python311Packages.langgraph-checkpoint-duckdb</li>
    <li>python311Packages.langgraph-checkpoint-duckdb.dist</li>
    <li>python311Packages.langgraph-checkpoint-postgres</li>
    <li>python311Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python311Packages.langgraph-checkpoint-sqlite</li>
    <li>python311Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python311Packages.langgraph-checkpoint.dist</li>
    <li>python311Packages.langgraph-sdk</li>
    <li>python311Packages.langgraph-sdk.dist</li>
    <li>python311Packages.langgraph.dist</li>
    <li>python312Packages.langgraph</li>
    <li>python312Packages.langgraph-checkpoint</li>
    <li>python312Packages.langgraph-checkpoint-duckdb</li>
    <li>python312Packages.langgraph-checkpoint-duckdb.dist</li>
    <li>python312Packages.langgraph-checkpoint-postgres</li>
    <li>python312Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python312Packages.langgraph-checkpoint-sqlite</li>
    <li>python312Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python312Packages.langgraph-checkpoint.dist</li>
    <li>python312Packages.langgraph-sdk</li>
    <li>python312Packages.langgraph-sdk.dist</li>
    <li>python312Packages.langgraph.dist</li>
  </ul>
</details>

**Comment by sarahec** [comment]
> What's the issue with aarch64-linux ?

Notice that it's all python 3.11 failures. I know that arch64 linux builds aren't as stable as the others. 

Also, I don't have an arch64-linux machine to test on, so if you want to look at the logs from yours it may shed some light.

**Comment by sarahec** [comment]
@GaetanLepage can you shed any light on the arch64-linux failures?

**Comment by sarahec** [comment]
@ofborg build python311Packages.langgraph-checkpoint-postgres

**Comment by GaetanLepage** [comment]
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 362648`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 24 packages built:</summary>
  <ul>
    <li>python311Packages.langgraph</li>
    <li>python311Packages.langgraph-checkpoint</li>
    <li>python311Packages.langgraph-checkpoint-duckdb</li>
    <li>python311Packages.langgraph-checkpoint-duckdb.dist</li>
    <li>python311Packages.langgraph-checkpoint-postgres</li>
    <li>python311Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python311Packages.langgraph-checkpoint-sqlite</li>
    <li>python311Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python311Packages.langgraph-checkpoint.dist</li>
    <li>python311Packages.langgraph-sdk</li>
    <li>python311Packages.langgraph-sdk.dist</li>
    <li>python311Packages.langgraph.dist</li>
    <li>python312Packages.langgraph</li>
    <li>python312Packages.langgraph-checkpoint</li>
    <li>python312Packages.langgraph-checkpoint-duckdb</li>
    <li>python312Packages.langgraph-checkpoint-duckdb.dist</li>
    <li>python312Packages.langgraph-checkpoint-postgres</li>
    <li>python312Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python312Packages.langgraph-checkpoint-sqlite</li>
    <li>python312Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python312Packages.langgraph-checkpoint.dist</li>
    <li>python312Packages.langgraph-sdk</li>
    <li>python312Packages.langgraph-sdk.dist</li>
    <li>python312Packages.langgraph.dist</li>
  </ul>
</details>

---
### `aarch64-linux`
<details>
  <summary>:white_check_mark: 24 packages built:</summary>
  <ul>
    <li>python311Packages.langgraph</li>
    <li>python311Packages.langgraph-checkpoint</li>
    <li>python311Packages.langgraph-checkpoint-duckdb</li>
    <li>python311Packages.langgraph-checkpoint-duckdb.dist</li>
    <li>python311Packages.langgraph-checkpoint-postgres</li>
    <li>python311Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python311Packages.langgraph-checkpoint-sqlite</li>
    <li>python311Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python311Packages.langgraph-checkpoint.dist</li>
    <li>python311Packages.langgraph-sdk</li>
    <li>python311Packages.langgraph-sdk.dist</li>
    <li>python311Packages.langgraph.dist</li>
    <li>python312Packages.langgraph</li>
    <li>python312Packages.langgraph-checkpoint</li>
    <li>python312Packages.langgraph-checkpoint-duckdb</li>
    <li>python312Packages.langgraph-checkpoint-duckdb.dist</li>
    <li>python312Packages.langgraph-checkpoint-postgres</li>
    <li>python312Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python312Packages.langgraph-checkpoint-sqlite</li>
    <li>python312Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python312Packages.langgraph-checkpoint.dist</li>
    <li>python312Packages.langgraph-sdk</li>
    <li>python312Packages.langgraph-sdk.dist</li>
    <li>python312Packages.langgraph.dist</li>
  </ul>
</details>

---
### `x86_64-darwin`
<details>
  <summary>:white_check_mark: 24 packages built:</summary>
  <ul>
    <li>python311Packages.langgraph</li>
    <li>python311Packages.langgraph-checkpoint</li>
    <li>python311Packages.langgraph-checkpoint-duckdb</li>
    <li>python311Packages.langgraph-checkpoint-duckdb.dist</li>
    <li>python311Packages.langgraph-checkpoint-postgres</li>
    <li>python311Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python311Packages.langgraph-checkpoint-sqlite</li>
    <li>python311Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python311Packages.langgraph-checkpoint.dist</li>
    <li>python311Packages.langgraph-sdk</li>
    <li>python311Packages.langgraph-sdk.dist</li>
    <li>python311Packages.langgraph.dist</li>
    <li>python312Packages.langgraph</li>
    <li>python312Packages.langgraph-checkpoint</li>
    <li>python312Packages.langgraph-checkpoint-duckdb</li>
    <li>python312Packages.langgraph-checkpoint-duckdb.dist</li>
    <li>python312Packages.langgraph-checkpoint-postgres</li>
    <li>python312Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python312Packages.langgraph-checkpoint-sqlite</li>
    <li>python312Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python312Packages.langgraph-checkpoint.dist</li>
    <li>python312Packages.langgraph-sdk</li>
    <li>python312Packages.langgraph-sdk.dist</li>
    <li>python312Packages.langgraph.dist</li>
  </ul>
</details>

---
### `aarch64-darwin`
<details>
  <summary>:white_check_mark: 24 packages built:</summary>
  <ul>
    <li>python311Packages.langgraph</li>
    <li>python311Packages.langgraph-checkpoint</li>
    <li>python311Packages.langgraph-checkpoint-duckdb</li>
    <li>python311Packages.langgraph-checkpoint-duckdb.dist</li>
    <li>python311Packages.langgraph-checkpoint-postgres</li>
    <li>python311Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python311Packages.langgraph-checkpoint-sqlite</li>
    <li>python311Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python311Packages.langgraph-checkpoint.dist</li>
    <li>python311Packages.langgraph-sdk</li>
    <li>python311Packages.langgraph-sdk.dist</li>
    <li>python311Packages.langgraph.dist</li>
    <li>python312Packages.langgraph</li>
    <li>python312Packages.langgraph-checkpoint</li>
    <li>python312Packages.langgraph-checkpoint-duckdb</li>
    <li>python312Packages.langgraph-checkpoint-duckdb.dist</li>
    <li>python312Packages.langgraph-checkpoint-postgres</li>
    <li>python312Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python312Packages.langgraph-checkpoint-sqlite</li>
    <li>python312Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python312Packages.langgraph-checkpoint.dist</li>
    <li>python312Packages.langgraph-sdk</li>
    <li>python312Packages.langgraph-sdk.dist</li>
    <li>python312Packages.langgraph.dist</li>
  </ul>
</details>

**Comment by GaetanLepage** [comment]
> @GaetanLepage can you shed any light on the aarch64-linux failures?

Only a flaky test. All good now !

**Review by GaetanLepage — CHANGES_REQUESTED** [review]
Could you take this opportunity to replace `rev = "refs/tags/TAG";` by `tag = "TAG";` in `fetchFromGitHub` please ?

**Comment by sarahec** [comment]
> Could you take this opportunity to replace `rev = "refs/tags/TAG";` by `tag = "TAG";` in `fetchFromGitHub` please ?

Done, as per #355973

**Comment by sarahec** [comment]
Those "rebuild-{darwin,linux} 101-500" tags don't seem right to me. The rebuild is 24 or 25 per platform * 2 architectures.

**Comment by sarahec** [comment]
I decided to try rebasing to see if that gets ofborg to relabel it correctly

**Comment by GaetanLepage** [comment]
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 362648`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 24 packages built:</summary>
  <ul>
    <li>python311Packages.langgraph</li>
    <li>python311Packages.langgraph-checkpoint</li>
    <li>python311Packages.langgraph-checkpoint-duckdb</li>
    <li>python311Packages.langgraph-checkpoint-duckdb.dist</li>
    <li>python311Packages.langgraph-checkpoint-postgres</li>
    <li>python311Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python311Packages.langgraph-checkpoint-sqlite</li>
    <li>python311Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python311Packages.langgraph-checkpoint.dist</li>
    <li>python311Packages.langgraph-sdk</li>
    <li>python311Packages.langgraph-sdk.dist</li>
    <li>python311Packages.langgraph.dist</li>
    <li>python312Packages.langgraph</li>
    <li>python312Packages.langgraph-checkpoint</li>
    <li>python312Packages.langgraph-checkpoint-duckdb</li>
    <li>python312Packages.langgraph-checkpoint-duckdb.dist</li>
    <li>python312Packages.langgraph-checkpoint-postgres</li>
    <li>python312Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python312Packages.langgraph-checkpoint-sqlite</li>
    <li>python312Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python312Packages.langgraph-checkpoint.dist</li>
    <li>python312Packages.langgraph-sdk</li>
    <li>python312Packages.langgraph-sdk.dist</li>
    <li>python312Packages.langgraph.dist</li>
  </ul>
</details>

---
### `aarch64-linux`
<details>
  <summary>:white_check_mark: 24 packages built:</summary>
  <ul>
    <li>python311Packages.langgraph</li>
    <li>python311Packages.langgraph-checkpoint</li>
    <li>python311Packages.langgraph-checkpoint-duckdb</li>
    <li>python311Packages.langgraph-checkpoint-duckdb.dist</li>
    <li>python311Packages.langgraph-checkpoint-postgres</li>
    <li>python311Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python311Packages.langgraph-checkpoint-sqlite</li>
    <li>python311Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python311Packages.langgraph-checkpoint.dist</li>
    <li>python311Packages.langgraph-sdk</li>
    <li>python311Packages.langgraph-sdk.dist</li>
    <li>python311Packages.langgraph.dist</li>
    <li>python312Packages.langgraph</li>
    <li>python312Packages.langgraph-checkpoint</li>
    <li>python312Packages.langgraph-checkpoint-duckdb</li>
    <li>python312Packages.langgraph-checkpoint-duckdb.dist</li>
    <li>python312Packages.langgraph-checkpoint-postgres</li>
    <li>python312Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python312Packages.langgraph-checkpoint-sqlite</li>
    <li>python312Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python312Packages.langgraph-checkpoint.dist</li>
    <li>python312Packages.langgraph-sdk</li>
    <li>python312Packages.langgraph-sdk.dist</li>
    <li>python312Packages.langgraph.dist</li>
  </ul>
</details>

---
### `x86_64-darwin`
<details>
  <summary>:white_check_mark: 24 packages built:</summary>
  <ul>
    <li>python311Packages.langgraph</li>
    <li>python311Packages.langgraph-checkpoint</li>
    <li>python311Packages.langgraph-checkpoint-duckdb</li>
    <li>python311Packages.langgraph-checkpoint-duckdb.dist</li>
    <li>python311Packages.langgraph-checkpoint-postgres</li>
    <li>python311Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python311Packages.langgraph-checkpoint-sqlite</li>
    <li>python311Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python311Packages.langgraph-checkpoint.dist</li>
    <li>python311Packages.langgraph-sdk</li>
    <li>python311Packages.langgraph-sdk.dist</li>
    <li>python311Packages.langgraph.dist</li>
    <li>python312Packages.langgraph</li>
    <li>python312Packages.langgraph-checkpoint</li>
    <li>python312Packages.langgraph-checkpoint-duckdb</li>
    <li>python312Packages.langgraph-checkpoint-duckdb.dist</li>
    <li>python312Packages.langgraph-checkpoint-postgres</li>
    <li>python312Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python312Packages.langgraph-checkpoint-sqlite</li>
    <li>python312Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python312Packages.langgraph-checkpoint.dist</li>
    <li>python312Packages.langgraph-sdk</li>
    <li>python312Packages.langgraph-sdk.dist</li>
    <li>python312Packages.langgraph.dist</li>
  </ul>
</details>

---
### `aarch64-darwin`
<details>
  <summary>:white_check_mark: 24 packages built:</summary>
  <ul>
    <li>python311Packages.langgraph</li>
    <li>python311Packages.langgraph-checkpoint</li>
    <li>python311Packages.langgraph-checkpoint-duckdb</li>
    <li>python311Packages.langgraph-checkpoint-duckdb.dist</li>
    <li>python311Packages.langgraph-checkpoint-postgres</li>
    <li>python311Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python311Packages.langgraph-checkpoint-sqlite</li>
    <li>python311Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python311Packages.langgraph-checkpoint.dist</li>
    <li>python311Packages.langgraph-sdk</li>
    <li>python311Packages.langgraph-sdk.dist</li>
    <li>python311Packages.langgraph.dist</li>
    <li>python312Packages.langgraph</li>
    <li>python312Packages.langgraph-checkpoint</li>
    <li>python312Packages.langgraph-checkpoint-duckdb</li>
    <li>python312Packages.langgraph-checkpoint-duckdb.dist</li>
    <li>python312Packages.langgraph-checkpoint-postgres</li>
    <li>python312Packages.langgraph-checkpoint-postgres.dist</li>
    <li>python312Packages.langgraph-checkpoint-sqlite</li>
    <li>python312Packages.langgraph-checkpoint-sqlite.dist</li>
    <li>python312Packages.langgraph-checkpoint.dist</li>
    <li>python312Packages.langgraph-sdk</li>
    <li>python312Packages.langgraph-sdk.dist</li>
    <li>python312Packages.langgraph.dist</li>
  </ul>
</details>

**Review by GaetanLepage — APPROVED** [review]
LGTM

**Comment by GaetanLepage** [comment]
> Those "rebuild-{darwin,linux} 101-500" tags don't seem right to me. The nixpkgs-review rebuild is 24 or 25 per platform * 2 architectures. Or is it a signal I need to rebase?

It is a bug introduced by me in https://github.com/NixOS/nixpkgs/pull/362844.
Fixed in https://github.com/NixOS/nixpkgs/pull/363851.

**Pull request merged**